src/index.css

```diff
@@ -55,4 +55,26 @@ div.code {
         border-top-left-radius: 4px;
         border-bottom-left-radius: 4px;
     }
+  
+  .sidebar-after {
+    content: "";
+    position: absolute;
+    left: 0px;
+    width: 4px;
+    height: 44px;
+    background-color: red;
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
+  }
+
+  .sidebar-before {
+    content: "";
+    position: absolute;
+    left: 0px;
+    width: 4px;
+    height: 44px;
+    background-color: rgb(254, 226, 226);
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
+  }
```

Comment:
I wonder what about those styles is unique to css and what could be reproduced in Tailwind. I imagine that when we have  already existing css to base on it is easy to just enter it, but for example I was also able to introduce some inline style for one line when I needed background https://github.com/mamali-fs/api-deriv-com-solidjs/blob/master/src/Pages/Home/Hero.tsx#L4 So in here I understand that tailwind gives as way for `before` and `after` classes but no way to specify before and after heights?

Reply:
In that case, I had to repeat hover:before: multiple times, so I used class instead😁